app/views/containers/show.html.erb

```diff
@@ -47,7 +47,7 @@
               <td><%= _('Environment Variables') %></td>
               <td>
                 <table id="environment_variables" class="table table-bordered" style="table-layout:fixed; word-wrap: break-word">
-                  <% @container.in_fog.attributes['config_env'].each do |environment_variable| %>
+                  <% @container.in_fog.environment_variables.each do |environment_variable| %>
```

Comment:
Can you also add some HTML for displaying the same kind of table for exposed ports? 
Bonus points for creating a helper for these kind of tables. :smile: